samples/synthesize.js

```diff
@@ -14,14 +14,23 @@
  */
 
 'use strict';
+function synthesizeSpeechAsync(request = {}) {
```

Comment:
make an async function

Reply:
removed and used in original functions itself to maintain require statements and client creations with in a respective functions